ParkingLot/src/main/java/models/ParkingSpots/TwoWheelerParkingSpot.java

```diff
@@ -1,16 +0,0 @@
-package models.ParkingSpots;
-
-import models.Vehicle;
-import models.VehicleType;
-
-public class TwoWheelerParkingSpot extends ParkingSpot {
-
-    public TwoWheelerParkingSpot() {
-        super(true, VehicleType.TWO_WHEELER, 10);
-    }
-
-    public int getPrice() {
```

Comment:
getPrice method is just calling the super.getPrice() in both the class (2wheeler and 4wheeler) why not pass this as an argument in constructor itself?

Reply:
Not sure if i get you, but we have passed the price in the constructor. The last argument (10) in line 9 of this file.